### ISSUE TITLE
Irregular inflections are incorrect

### ISSUE THREAD

**Issue by dovej** [mvysny/aedict#758]
The current inflections pane seems to run a blind algorithm. As a result, incorrect inflections are displayed for irregular verbs. For example, する→しって, 行く→行いて, etc. くる seems to be special cased, but that's the only one.

Meanwhile, the entries for いい and かっこいい are missing inflections entirely.

While there are a relatively small set of irregularities in Japanese, making exhaustive special-casing feasible, it is worth pointing out that the JMDict web client has conjugation tables for all of the irregular words and gets them right. It is probably safer and easier to source the base conjugation data from there. More advanced grammatical patterns like 「〜た方がいい」could be constructed by building off the base conjugation data.

JMDict's conjugation for する: http://nihongo.monash.edu/cgi-bin/wwwjdic?1W%B0%D9%A4%EB_vs-i

**Comment by dovej** [comment]
Hmm... JMDict seems to punt on v5aru like くださる: http://nihongo.monash.edu/cgi-bin/wwwjdic?1W%B2%BC%A4%B5%A4%EB_v5aru The phrasing on that page suggests that it might be supported in the future, which is a point in favor of sourcing conjugation data from JMDict if possible (for the sake of updates).

However, the app Jsho **does** conjugate v5aru verbs, and based on my very elementary knowledge, it seems to get it right. But I don't know enough to be positive.

**Comment by mvysny** [comment]
Yup, that is true - kuru was the only one inflected in a special way. I have fixed suru #757 , let's fix iku next: http://www.sljfaq.org/cgi/verb-inflect.cgi?verb=iku

**Comment by mvysny** [comment]
行く fixed, let's fix v5aru like 下さる: https://en.wikipedia.org/wiki/Japanese_verb_conjugation

**Comment by mvysny** [comment]
v5aru fixed; I looked on いい and it's not adj-i nor adj-na so that's why the inflections are missing. I'll add a special inflector for that one.
